core-services/prow/02_config/openshift/cli-manager/_prowconfig.yaml

```diff
@@ -1,5 +1,70 @@
 tide:
   queries:
+  - includedBranches:
+    - openshift-4.10
+    - openshift-4.11
+    - openshift-4.12
+    - openshift-4.13
+    - openshift-4.14
+    - openshift-4.15
+    - openshift-4.16
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.0
+    - release-4.10
+    - release-4.11
+    - release-4.12
+    - release-4.13
+    - release-4.14
+    - release-4.15
+    - release-4.16
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
+    labels:
+    - approved
+    - backport-risk-assessed
+    - cherry-pick-approved
+    - jira/valid-bug
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - needs-rebase
+    repos:
+    - openshift/cli-manager
+  - includedBranches:
```

Comment:
This part looks to be exactly the same as for the previous `includedBranches`. Should be safe to move `openshift-4.17` and `release-4.17` there. The same for CLMO configuration.